extensions/ql-vscode/src/view/common/Dropdown.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import { ChangeEvent } from "react";
+import styled from "styled-components";
+
+const StyledDropdown = styled.select`
+  width: 100%;
+  height: calc(var(--input-height) * 1px);
+  background: var(--dropdown-background);
+  color: var(--foreground);
+  font-family: var(--font-family);
+  border: none;
+  padding: 2px 6px 2px 8px;
+`;
+
+type Props = {
+  value: string | undefined | false;
```

Comment:
Including `| false` here is kinda nice because it means in `MethodRow.tsx` we can pass in `showInputCell && modeledMethod?.input` instead of having to do `showInputCell ?modeledMethod?.input : undefined`. But this isn't essential and I'm happy to change the type to just be `string | undefined` if that's preferable.

Reply:
Although, I just realised we can do this logic of hiding the value when the input is disabled in the `Dropdown` component. So ignore the previous comment. The type can be simpler and we can just ignore the value when the input is disabled.